ros2_ws/src/rcdt_sensors/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ find_package(realsense2_camera_msgs REQUIRED)
 # Install nodes
```

Comment:
Ik zou hierboven ook nog een package dependency op rcdt_detection verwachten. We gebruiken namelijk de ros_to_cv2 en andersom van rcdt_detection.

Al moet ik zeggen, nu we heb breder gaan gebruiken moet dit misschien we naar rcdt_utilities verplaatsen